src/main/java/frc/robot/subsystems/gripper/Robot.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.gripper;
+
+import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+
+public class Robot extends TimedRobot {
+
+  private Gripper gripper;
+
+  @Override
+  public void robotInit() {
+    gripper = Gripper.getInstance();
+    gripper.configureDashboard();
+  }
+
+  @Override
+  public void robotPeriodic() {
+    gripper.periodic();
+    CommandScheduler.getInstance().run();
+  }
+
+  @Override
+  public void disabledInit() {}
```

Comment:
put the second curly brace on the line after this (do this for the other methods as well)